### PR TITLE
fix position number in storeTemplate

### PR DESCRIPTION
**Pull request by anhquan223241**
correct positionNumber is "(256 \* page) + i "

### CODE REVIEW

**Comment by philippmeisberger** [comment]
You are right. My mistake ;) Bastian should merge it

File: src/files/pyfingerprint/pyfingerprint.py [hunk 1]
```diff
@@ -795,7 +795,7 @@ def storeTemplate(self, positionNumber = -1, charBufferNumber = 0x01):
                 for i in range(0, len(templateIndex)):
                     ## Index not used?
                     if ( templateIndex[i] == False ):
-                        positionNumber = i
+                        positionNumber = (256 * page) + i
```
Comment (philippmeisberger) [inline src/files/pyfingerprint/pyfingerprint.py]: This fix is better: `positionNumber = (len(templateIndex) * page) + i`
There might be another page capacity in a later version ;)

Comment (bastianraschke) [reply src/files/pyfingerprint/pyfingerprint.py]: @anhquan223241 It would be nice if you can improve your pull request, so i can merge it :)

Comment (anhquan223241) [reply src/files/pyfingerprint/pyfingerprint.py]: Ok I changed it ;)

Comment (bastianraschke) [reply src/files/pyfingerprint/pyfingerprint.py]: Perfect, thanks :)

**Pull request merged**